portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,65 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
+.site-background {
+  background-color: #94B4CB;
+  font-family: monospace;
+}
+
+.header {
+  font-size: 300%;
+  margin-left: 30%;
+  margin-top: 20%;
+}
+
+ul {
+  list-style-type: none;
+  margin-left: 27.75%;
+}
+
+li {
+  float: left;
+  padding-right: 20px;
+  padding-left: 20px;
+}
+
+li a {
+  background-color: #9FA0C3;
+  border-bottom-color: #3C617C;
+  border-bottom-style: solid;
+  display: block;
+  color: black;
```

Comment:
It's good style to sort the rules within styles alphabetically.  You've mostly been doing it already, but it's not sorted here.

Reply:
whoops I thought I did this for all of them!